File: common/src/main/java/me/pandamods/pandalib/PandaLibClient.java
package me.pandamods.pandalib;

import me.pandamods.test.PandaLibClientTest;
import me.pandamods.test.PandaLibTest;
import net.fabricmc.api.EnvType;
import net.fabricmc.api.Environment;

@Environment(EnvType.CLIENT)
public class PandaLibClient {
	public static void init() {
		if (PandaLibTest.shouldInit()) {
			PandaLibClientTest.init();
		}
	}
}

File: src/main/kotlin/dev/pandasystems/pandalib/client/PandaLibClient.kt


package dev.pandasystems.pandalib.client

import dev.pandasystems.pandalib.core.client.event.EventHandlerClient
import net.fabricmc.api.EnvType
import net.fabricmc.api.Environment

@Environment(EnvType.CLIENT)
object PandaLibClient {
	@JvmStatic
	fun init() {
		EventHandlerClient.init()
	}
}